### ISSUE TITLE
Use translations database

### ISSUE THREAD

**Issue by hamedali1336** [quran/quran_android#2804]
Can I use translations databases in my app ?

**Comment by ahmedre** [comment]
Ideally get the translations from quranenc.com which is where we get most of ours. They also tell you when they were last updated, etc and allow you to submit error reports directly on the ayah.

**Comment by hamedali1336** [comment]
Ok , thanks
But tafseer like baghawy.db and ibnkathir.db not in this site

**Comment by ahmedre** [comment]
Yes these are used with permission of tafsir.app but unfortunately have many typos.

**Comment by hamedali1336** [comment]
I obtained the link of these databases from this file 
https://github.com/quran/quran_android/blob/9d99a3f328c91a3dd0ceda916f9e61fa73d66471/pages/madani/src/main/java/com/quran/data/page/provider/madani/MadaniPageProvider.kt#L35
Can I use it to download them in runtime ?

**Comment by hamedali1336** [comment]
I manage to get download url 
Like this 
Https://android.quran.com/data/getTranslation.php?id=58&ext=zip
Can I use them to download at runtime ?

**Comment by Sayarhtoo1** [comment]
plz add Burmese(Myanmar) Translation and indopak colour scripts.
